src/components/form/Checkbox/Checkbox.tsx

```diff
@@ -141,9 +133,29 @@ const Container = styled.div`
 `
 
 const Main = styled.div`
+  align-items: flex-start;
+  display: inline-flex;
+  align-items: center;
+  vertical-align: middle;
+  margin-left: -11px;
+  margin-right: 16px;
+  margin-left: 0;
+  margin-right: 0;
+  -webkit-align-items: flex-start;
+  -webkit-box-align: flex-start;
+  -ms-flex-align: flex-start;
```

Comment:
why do you need to prefix these properties?

Reply:
not sure, will remove, also clean duplicated rules!